cmd/ddev/cmd/exec.go

```diff
@@ -75,6 +73,22 @@ ddev exec --raw -- ls -lR`,
 	},
 }
 
+// quoteArgs quotes any arguments that contain spaces.
```

Comment:
It may be worth looking at the quoteArgs() used internally in go:
```
func quoteArgs(args []string) string {
	var b strings.Builder
	for i, arg := range args {
		if i > 0 {
			b.WriteString(" ")
		}
		if strings.ContainsAny(arg, "'"+argSepChars) {
			// Quote the argument to a form that would be parsed as a single argument.
			b.WriteString("'")
			b.WriteString(strings.ReplaceAll(arg, "'", "''"))
			b.WriteString("'")
		} else {
			b.WriteString(arg)
		}
	}
	return b.String()
}
```

Reply:
Thank you!

I didn't know about this function, I'll copy it.